docs/source/traits_user_manual/listening.rst

```diff
@@ -494,7 +494,7 @@ To notify about changes to any trait attribute on a class, define a method
 named _anytrait_changed().
 
 .. index::
-   pair: examples; _any_trait_changed()
+   pair: examples; _anytrait_changed()
```

Comment:
This one is an actual doc bug - the example is demonstrating use of the `_anytrait_changed` method. There is no `_any_trait_changed` method.